tests/managers/account_tests.py

```diff
@@ -176,3 +176,7 @@ def test_get_bandwidth_pool_counts(self):
         total = self.manager.get_bandwidth_pool_counts(1234)
         self.assert_called_with('SoftLayer_Network_Bandwidth_Version1_Allotment', 'getObject', identifier=1234)
         self.assertEqual(total, 2)
+
+    def test_get_dedicated_hosts(self):
+        self.manager.get_dedicated_hosts("", "")
```

Comment:
If we have a default data for this function, the call will easier
```suggestion
        self.manager.get_dedicated_hosts()
```

Reply:
Done